packages/account/src/test-utils/launchNode.test.ts

```diff
@@ -31,7 +32,7 @@ function mockSpawn(params: { shouldError: boolean } = { shouldError: false }) {
         // The `Binding GraphQL provider to` message simulates a fuel-core
         // successful startup log message, usually meaning that the node
         // is up and waiting for connections
-        fn('Binding GraphQL provider to');
+        fn('Binding GraphQL provider to 0.0.0.0:4000');
```

Comment:
Why is this bound to a specific port now?

Reply:
It's because `launchNode` is now [reading](https://github.com/FuelLabs/fuels-ts/pull/1356/files#diff-b8f07cac51692505e88e065262fd373293a373ed5f6196eb13ec811e37aee136R241) from the logs to get the IP and port. Previously, the string `Binding GraphQL provider to` was used as a mock message that'd cause `launchNode` to return successfully; now we need the `0.0.0.0:4000`, otherwise the logic would fail.

We're now reading the IP and port from the logs because we're using the `0` port [here](https://github.com/FuelLabs/fuels-ts/pull/1356/files#diff-3c0413287e9234f6662e455d68cdf5c2daf63406cd32c3b392f53c6fd5cd525fR68), thereby asking the OS to assign a port to our test node.